aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -55,16 +47,16 @@ module aptos_token::token_v1 {
 
     struct Token has store {
         id: TokenId,
-        // the amount of tokens. Only serial_number = 0 can have a value bigger than 1.
-        value: u64,
+        // the amount of tokens. Only property_version = 0 can have a value bigger than 1.
+        amount: u64,
     }
 
     /// global unique identifier of a token
     struct TokenId has store, copy, drop {
-        // the id to the common token data shared by token with different serial number
+        // the id to the common token data shared by token with different property_version
         token_data_id: TokenDataId,
-        // the serial_number of a token. Token with dfiferent serial number can have different value of PropertyMap
-        serial_number: u64,
+        // the property_version of a token. Token with dfiferent property_version can have different value of PropertyMap
+        property_version: u64,
```

Comment:
I found it hard to explain using serial_number to @hariria

Reply:
It's weird because fungible tokens change their serial numbers if their property mutates whilst a serial number is supposed to be immutable like a guid.

Reply:
Like your iPhone's serial no. doesn't change if you change something about it